src/coreclr/src/vm/jitinterface.cpp

```diff
@@ -12362,6 +12386,11 @@ CorJitResult CallCompileMethodWithSEHWrapper(EEJitManager *jitMgr,
          }
     }
 
+#if !defined(TARGET_X86) || !defined(TARGET_WINDOWS)
```

Comment:
The equivalent code in crossgen2 that sets `CORJIT_FLAG_REVERSE_PINVOKE` is not under this conditon. 

It may be best to fail compilation of the native callable methods in both crossgen and crossgen2.

Reply:
(so that we do not have a problem with ABI mismatch once the Windows x86 path is done properly)